app/redhat_insights/default/data/ui/views/main_dashboard.xml

```diff
@@ -1154,18 +1142,18 @@
 					"earliest": "-30d",
 					"latest": "now"
 				},
-				"query": "index=redhatinsights \n| stats count by application\n| eval application = case(\n      application==\"malware-detection\", \"Malware\",\n      !isnull(application), upper(substr(application,1,1)).substr(application,2)\n      )\n| rename application as Application, count as Count"
+				"query": "index=redhatinsights \n| stats count by application\n| eval application = case(\n      application==\"malware-detection\", \"Malware\",\n      application==\"resource-optimization\", \"ROS\",\n      !isnull(application), upper(substr(application,1,1)).substr(application,2)\n      )\n| rename application as Application, count as Count"
 			},
 			"name": "Search_Pie_All_Events_30d"
 		},
 		"ds_TbymX4BM_ds_9hVCveWJ_ds_A28QM2S4_ds_snKvID23": {
 			"type": "ds.search",
 			"options": {
 				"queryParameters": {
-					"earliest": "-48h",
+					"earliest": "-30d@d",
```

Comment:
Shouldn't it be `-30d`?

The query specifies `earliest=-30d latest=now()`. AFAIK the query parameters have precedence over `options.queryParameters`.